src/fusor/fusion_matching.py

```diff
@@ -0,0 +1,266 @@
+"""Module for matching assayed fusions against categorical fusions"""
+
+import pickle
+from pathlib import Path
+
+from civicpy import civic
+
+from fusor.harvester import CIVICHarvester
+from fusor.models import (
+    AssayedFusion,
+    CategoricalFusion,
+    GeneElement,
+    LinkerElement,
+    MultiplePossibleGenesElement,
+    TranscriptSegmentElement,
+    UnknownGeneElement,
+)
+from fusor.translator import Translator
+
+
+class FusionMatcher:
+    """Class for matching assayed fusions against categorical fusions"""
+
+    def __init__(self, translator: Translator) -> None:
+        """Initialize Translator class
+
+        :param translator: A Translator instance
+        :return None
+        """
+        self.translator = translator
+
+    async def generate_civic_categorical_fusions(self) -> list[CategoricalFusion]:
+        """Load fusion variants from CIViC and convert to CategoricalFusion objects
+
+        :return A list of CategoricalFusion objects
+        """
+        # Load in all accepted fusions variants
+        variants = civic.get_all_fusion_variants(include_status="accepted")
+        harvester = CIVICHarvester(fusions_list=variants)
+        fusions_list = harvester.load_records()
+
+        categorical_fusions = []
+        for fusion in fusions_list:
+            if "?" in fusion.vicc_compliant_name:
+                continue
+            cex = await self.translator.from_civic(civic=fusion)
+            categorical_fusions.append(cex)
+        return categorical_fusions
+
+    def save_civic_categorical_fusions(
```

Comment:
I'm not seeing this being used anywhere? Did you need to provide an example of this in your notebook?